renderlib/AppScene.h

```diff
@@ -35,6 +36,7 @@ struct VolumeDisplay
   float m_opacity[MAX_CPU_CHANNELS];
   bool m_enabled[MAX_CPU_CHANNELS];
   float m_labels[MAX_CPU_CHANNELS];
+  ColorRamp m_colormap[MAX_CPU_CHANNELS];
```

Comment:
one color ramp for each channel, now stored in Scene but formerly were stored in ImageXYZC::Channelu16